uaclient/tests/test_contract.py

```diff
@@ -49,8 +49,8 @@ class TestUAContractClient:
         "machine_id_response", (("contract-machine-id"), None)
     )
     @pytest.mark.parametrize(
-        "detach,expected_http_method",
-        ((None, "POST"), (False, "POST"), (True, "DELETE")),
+        "expected_http_method",
+        (("POST"), ("POST")),
```

Comment:
the value no longer changes so I don't think we need this `@pytest.mark.parametrize` anymore
we can just hardcode "POST" in the body of the test

Reply:
Yes, on it.